Index: branches/qa/sites/all/modules/CMS/apachesolr/Solr_Base_Query.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- branches/qa/sites/all/modules/CMS/apachesolr/Solr_Base_Query.php	(revision )
+++ branches/qa/sites/all/modules/CMS/apachesolr/Solr_Base_Query.php	(revision 4540)
@@ -10,6 +10,7 @@
     $name = preg_quote($name, '/');
     // Range queries.  The "TO" is case-sensitive.
     $patterns[] = '/(^| |-)'. $name .':([\[\{](\S+) TO (\S+)[\]\}])/';
+	$patterns[] = '/(^| |-)'. $name .':([\[\{](\S+) (\S+)[\]\}])/';
     // Match quoted values.
     $patterns[] = '/(^| |-)'. $name .':"([^"]*)"/';
     // Match unquoted values.
Index: branches/qa/sites/all/modules/CMS/apachesolr/apachesolr_search.module
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- branches/qa/sites/all/modules/CMS/apachesolr/apachesolr_search.module	(revision )
+++ branches/qa/sites/all/modules/CMS/apachesolr/apachesolr_search.module	(revision 4540)
@@ -186,11 +186,11 @@
 }
 
 function apachesolr_search_noresults() {
-      return t('<ul>
+      return t('<div class="box"><div class="content"><ul>
 <li>Check if your spelling is correct, or try removing filters.</li>
 <li>Remove quotes around phrases to match each word individually: <em>"blue drop"</em> will match less than <em>blue drop</em>.</li>
 <li>You can require or exclude terms using + and -: <em>big +blue drop</em> will require a match on <em>blue</em> while <em>big blue -drop</em> will exclude results that contain <em>drop</em>.</li>
-</ul>');
+</ul></div></div>');
 
 }
 
Index: branches/qa/sites/all/modules/CMS/apachesolr/apachesolr.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- branches/qa/sites/all/modules/CMS/apachesolr/apachesolr.js	(revision )
+++ branches/qa/sites/all/modules/CMS/apachesolr/apachesolr.js	(revision 4540)
@@ -4,10 +4,12 @@
   $('<a href="#" class="apachesolr-showhide"></a>').text(Drupal.t('Show more')).click(function() {
     if ($(this).parent().find('.apachesolr-hidden-facet:visible').length == 0) {
       $(this).parent().find('.apachesolr-hidden-facet').show();
+      $(this).attr('title','Show fewer');
       $(this).text(Drupal.t('Show fewer'));
     }
     else {
       $(this).parent().find('.apachesolr-hidden-facet').hide();
+      $(this).attr('title','Show more');
       $(this).text(Drupal.t('Show more'));
     }
     return false;
Index: branches/qa/sites/all/modules/CMS/apachesolr/apachesolr.module
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- branches/qa/sites/all/modules/CMS/apachesolr/apachesolr.module	(revision )
+++ branches/qa/sites/all/modules/CMS/apachesolr/apachesolr.module	(revision 4540)
@@ -1805,7 +1805,7 @@
   );
 
   // Don't need this, and just to be safe.
-  unset($options['attributes']['title']);
+ // unset($options['attributes']['title']);
   // Double encode + characters for clean URL Apache quirks.
   if (variable_get('clean_url', '0')) {
     $path = str_replace('+', '%2B', $path);
@@ -1837,6 +1837,7 @@
     $icon = ' '. theme('tablesort_indicator', $direction);
   }
   if ($active) {
+      $options['attributes']['title']=$text;
     if (isset($options['attributes']['class'])) {
       $options['attributes']['class'] .= ' active';
     }
