Index: password_policy.module
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- password_policy.module	(revision )
+++ password_policy.module	(revision 7171)
@@ -352,9 +352,17 @@
     case "user_profile_form":
     case "user_register":
       // Password change form.
-      $uid = isset($form['#uid']) ? $form['#uid'] : NULL;
-      //if ($uid == 1 && !variable_get('password_policy_admin', 0)) { break; }
-      $roles = isset($form['_account']['#value']) ? array_keys($form['_account']['#value']->roles) : array();
+
+      /* don't read user values from $form
+       * $uid = isset($form['#uid']) ? $form['#uid'] : NULL;
+       * //if ($uid == 1 && !variable_get('password_policy_admin', 0)) { break; }
+       * $roles = isset($form['_account']['#value']) ? array_keys($form['_account']['#value']->roles) : array();
+       */
+      // use Drupal $user object for user properties
+      global $user;
+      $uid = $user->uid;
+      $roles = array_keys(user_roles());
+
       $policy = _password_policy_load_active_policy($roles);
       $translate = array();
       if (!empty($policy['policy'])) {
@@ -367,7 +375,7 @@
 
       // Printing out the restrictions.
       if (variable_get('password_policy_show_restrictions', 0) && !empty($translate)) {
-        $restriction_html = '<div id="account-pass-restrictions"><ul><li>'. implode('</li><li>', $translate) .'</li></ul></div>';
+          $restriction_html = '<div id="account-pass-restrictions">'. theme('item_list', $translate, t('Password Requirements')) .'</div>';
         if (isset($form['account']) && is_array($form['account'])) {
           $form['account']['pass']['#prefix'] = $restriction_html;
         }
